Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Rating.hs

```diff
@@ -54,8 +60,20 @@ handler req ride = do
         "Updating existing rating for " +|| ride.id ||+ " with new rating " +|| ratingValue ||+ "."
       Esq.runTransaction $ do
         QRating.updateRating rideRating.id driverId ratingValue feedbackDetails
+  addFeedbackToTable feedbackChips ride.id driverId
   calculateAverageRating driverId
 
+addFeedbackToTable :: Maybe [String] -> Id DRide.Ride -> Id DP.Person -> Flow ()
+addFeedbackToTable feedbackChipsList rideId driverId = do
+  case feedbackChipsList of
+    Just feedbackChip -> do
+      traverse_ saveFeedback feedbackChip
+    Nothing -> return ()
+  where
+    saveFeedback feedbackItem = do
+      newFeedback <- buildFeedback rideId driverId (DT.pack feedbackItem)
+      Esq.runTransaction $ QFeedback.create newFeedback
```

Comment:
You create each item in separate transaction. Can you refactor, so that all items will be created n single transaction, like

```
    Esq.runTransaction $ do
      QFeedback.createMany newFeedbacks
```

Ideally all write queries should be within single transaction

Reply:
done